docs/appendices/release-notes/5.10.1.rst

```diff
@@ -144,3 +144,13 @@ Fixes
   by the :ref:`sql-restore-snapshot` statement when only a concrete table was
   specified as to be restored. Only the partitioned table definition was falsely
   restored, but not the the actual data.
+
+- Fixed an issue that would lead to wrong results for ``LEFT JOIN`` queries when
+  the join condition has a :ref:`CASE<scalar-case-when-then-end>` with an
```

Comment:
Minor: shouldn't we mention that it's regression in 5.10.0?

Reply:
yes, thx, I'm adding it.